src/vorta/keyring/secretstorage.py

```diff
@@ -18,7 +21,26 @@ def __init__(self):
         asyncio.set_event_loop(asyncio.new_event_loop())
         secretstorage.get_default_collection(self.connection)
 
+    def make_label(self, repo_url: str) -> str:
```

Comment:
Do we really need this function, given it's only used once?

Reply:
I think so, since it will allow future edits to access the label construction mechanism from a central place. This improves maintainability in comparison to multiple methods constructing the label on their own. If one wanted to change the label construction, all the lines constructing it would have to be found and changed. Now only this method has to be reimplemented.